lighthouse-core/test/audits/metrics/largest-contentful-paint-test.js

```diff
@@ -27,20 +31,53 @@ function generateContext({throttlingMethod}) {
 
 describe('Performance: largest-contentful-paint audit', () => {
   it('adjusts scoring based on form factor', async () => {
-    const artifactsMobile = generateArtifacts({trace, devtoolsLog, TestedAsMobileDevice: true});
+    const artifactsMobile = generateArtifacts({
+      trace,
+      devtoolsLog,
+      TestedAsMobileDevice: true,
+      HostUserAgent: 'Chrome/78.0.3904.70',
```

Comment:
probably should use a full UA string, just so it better matches what the real input will be.

e.g if we ever move to a centralized UA string parser function and it used a regex that does multiple capture groups on the OS and whatever else so would only match against the whole string, ideally these tests would keep passing without having to be updated.

```suggestion
      HostUserAgent: 'Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.70 Mobile Safari/537.36 Chrome-Lighthouse',
```
(and for the other `HostUserAgent`s below)